libs/server-sdk/README.md

```diff
@@ -102,6 +102,34 @@ for consistency across SDKs, as well as test networking behavior in a long-runni
 method in the SDK, and verify that event sending, flag evaluation, stream reconnection, and other aspects of the SDK all
 behave correctly.
 
+Validating SDK packages with the SLSA framework
+-------
+
+LaunchDarkly uses the [SLSA framework](https://slsa.dev/spec/v1.0/about) to help developers make their supply chain more secure by ensuring the authenticity of our published SDK packages. As part of [SLSA requirements for level 3 compliance](https://slsa.dev/spec/v1.0/requirements), LaunchDarkly publishes provenance about our SDK package builds using [Github's generic SLSA3 provenance generator](https://github.com/slsa-framework/slsa-github-generator/blob/main/internal/builders/generic/README.md#generation-of-slsa3-provenance-for-arbitrary-projects) for distribution alongside our packages. 
+
+The SLSA framework specifies some [recommendations for verifying build artifacts](https://slsa.dev/spec/v1.0/verifying-artifacts) in their documentation. For our C/C++ SDKs, we recommend the following steps:
```

Comment:
Is there a tool that can do these steps automatically, given a provenance file? Or some of them?